app/controllers/sessions_controller.rb

```diff
@@ -18,8 +18,13 @@ def destroy
 
   private
   def login_redirect user
-    log_in user
-    params[:session][:remember_me] == Settings.checked ? remember(user) : forget(user)
-    redirect_back_or user
+    if user.activated?
+      log_in user
+      params[:session][:remember_me] == Settings.checked ? remember(user) : forget(user)
+      redirect_back_or user
+    else
+      flash[:danger] = t("account_inactivated") << t("check_email_announce")
```

Comment:
t("account_inactivated") << t("check_email_announce") 
syntaxx này là sao em

Reply:
Là ghép 2 string lại với nhau ạ. Lí do không dùng dấu "+" vì dùng toán tử này sẽ tạo ra một biến tạm rồi mới gán lại cho message. Còn toán tử này thì không.